src/crimson/os/seastore/transaction_manager.h

```diff
@@ -433,7 +433,7 @@ class TransactionManager : public ExtentCallbackInterface {
     ceph_assert(total_remap_len < original_len);
 #endif
 
-    // FIXME: paddr can be absolute and pending
+    // Note that overwriting a pending extent shouldn't involve remapping
     ceph_assert(pin->get_val().is_absolute());
```

Comment:
See https://github.com/ceph/ceph/pull/55888#discussion_r1510637600

This assert looks less meaningful, how about updating it to comment?
```
// The according extent might be stable or pending.
```

Reply:
Forget to drop the ceph-assert?